src/components/FAQ.css

```diff
@@ -0,0 +1,96 @@
+.faq-container {
+    background-image: url(../images/backgrounds/FAQ_Bg.png);
```

Comment:
Set more background css settings for how picture loads onto screen. Each browsers default settings are different.